docs/src/use/command-line-interface.md

```diff
@@ -325,7 +325,7 @@ For example:
 
 {{ npx_tabs ({
     package: "eslint",
-    args: ["--config", "~/personal-eslintrc.js", "\\", "--resolve-plugins-relative-to", "/usr/local/lib/"]
+    args: ["--config", "~/personal-eslintrc.js", "--resolve-plugins-relative-to", "/usr/local/lib/"]
```

Comment:
A trailing backslash is not useful for commands that are contained within a single line.